SIT.Manager/Views/MainView.axaml

```diff
@@ -90,6 +90,7 @@
 			</ui:NavigationView.MenuItems>
 
 			<ui:NavigationView.FooterMenuItems>
+				<ui:NavigationViewItem IconSource="Settings" Content="{DynamicResource LinuxSettingsPageTitle}" Tag="SIT.Manager.Views.LinuxSettingsPage" IsVisible="{OnPlatform Default=False, Linux=True}"/>
 				<ui:NavigationViewItem IconSource="Settings" Content="{DynamicResource SettingsTitle}" Tag="SIT.Manager.Views.SettingsPage"/>
```

Comment:
I felt like it would clutter the settings page (especially since the default window size isn't that big).
I also feel like having a separate button makes it a bit easier to find for the user so they know they might want to check it out, instead of quickly scrolling through the otherwise long list of settings.
Merging the linux settings page back into the settings shouldn't be too difficult, however, so if you'd prefer that. The call is yours to make.

Reply:
You're right it would make it far too cluttered, I didn't explain what I meant to well. 

Excuse the roughness of these two examples I just did them very quickly but should hopefully show what I was meaning:

Either something along the lines of thise:
![image](https://github.com/stayintarkov/SIT.Manager.Avalonia/assets/112902041/f2ebe807-2ee9-4ce8-b74f-c1f3c2974309)


Or what I think would work better is splitting the whole settings page out into something more like this:
![image](https://github.com/stayintarkov/SIT.Manager.Avalonia/assets/112902041/d296852c-561c-4e39-8449-ba284cfe8efd)

Reply:
That looks very slick indeed

Reply:
The more I look at it the more I like this one, it just needs a bit of work to make it look properly nice but I think that'd work well: 
![322460309-d296852c-561c-4e39-8449-ba284cfe8efd](https://github.com/stayintarkov/SIT.Manager.Avalonia/assets/112902041/3610a892-c5cd-48d1-9787-ad657b7561be)

You can do it if you want but we can just leave the way you've done the Linux settings page for now and can change all the settings to look this way in a separate PR.

Reply:
I'm not the best front-end guy so I think it's better to leave that to someone else :sweat_smile: